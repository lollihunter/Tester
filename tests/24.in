20 2 20
1 1
14 1
11 2
2 2
8 1
2 1
4 1
14 2
15 2
20 1
13 1
17 2
5 1
7 2
9 1
3 2
4 2
18 1
11 1
20 2
