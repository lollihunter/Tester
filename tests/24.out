15
1 6 16 17 13 14 5 15 19 11 2 9 12 18 20 
